mock/server/worker.go

```diff
@@ -199,3 +206,13 @@ func registerToken(c *gin.Context) {
 
 	c.JSON(http.StatusCreated, body)
 }
+
+// getQueueCreds returns mock JSON for a http POST.
+func getQueueCreds(c *gin.Context) {
```

Comment:
This mock should have a fail option for testing. Example: https://github.com/go-vela/server/blob/7ee3ae39552c14a2eb7777195c23a15cd2bf4059/mock/server/worker.go#L181-L201